lib/mirage.ml

```diff
@@ -729,13 +729,13 @@ let ipv4_qubes db ethernet arp = ipv4_qubes_conf $ db $ ethernet $ arp
 
 let ipv6_conf ?addresses ?netmasks ?gateways () = impl @@ object
     inherit base_configurable
-    method ty = ethernet @-> time @-> mclock @-> ipv6
+    method ty = ethernet @-> random @-> time @-> mclock @-> ipv6
     method name = Name.create "ipv6" ~prefix:"ipv6"
     method module_name = "Ipv6.Make"
     method packages = Key.pure [ package ~sublibs:["ipv6"] "tcpip" ]
```

Comment:
My plan was:
0) merge `make depends` fix to mirage
1) constrain mirage 3.0.0 to tcpip = 3.0.0
2) release tcpip 3.1.0 with mtu, improved random in ipv6, bugfixes (tcpip is already constrained to mirage-protocols* >= 1.1.0)
3) release mirage 3.0.1 depending on tcpip 3.1.0 (+mirage-protocols, mirage-protocols-lwt 1.1.0), and constraining the minimum tcpip version in generated unikernels to 3.1.0

I intend to do this today so not sure it's worth reviving mirage-dev, but if I take longer than that it probably is.